addons/mail/static/tests/helpers/mock_server/models/mail_thread.js

```diff
@@ -322,9 +306,6 @@ patch(MockServer.prototype, {
                         message: Object.assign(messageFormat, { temporary_id }),
                     },
                 ]);
-                if (message.author_id === this.pyEnv.currentPartnerId) {
-                    this._mockDiscussChannel_ChannelSeen(ids, message.id);
-                }
```

Comment:
This might be a simplification for what is done in `_message_post_after_hook`, I'm not sure if we have that method in JS.